R/bundle.R

```diff
@@ -145,6 +184,7 @@ createAppManifest <- function(appDir, files, users) {
   # create the manifest
   manifest <- list()
   manifest$version <- 1
+  manifest$appmode <- appMode
```

Comment:
FWIW shinyapps uses schema validation for manifests. I believe this will break deploying to shinyapps.